Sources/Commands/SwiftTestTool.swift

```diff
@@ -837,20 +861,27 @@ extension SwiftTestTool: ToolName {
 
 /// Creates the environment needed to test related tools.
 fileprivate func constructTestEnvironment(
-    sanitizers: EnabledSanitizers,
-    toolchain: UserToolchain
+    toolchain: UserToolchain,
+    options: ToolOptions,
+    buildParameters: BuildParameters
 ) throws -> [String: String] {
+    let sanitizers = options.sanitizers
+
     // This only makes sense for macOS.
   #if !os(macOS)
     return Process.env
   #else
+    var env = Process.env
+    if options.shouldEnableCodeCoverage {
+        let coverageDir = buildParameters.buildPath.appending(component: "codecoverage")
```

Comment:
Lets call the directory `codecov` and the filename can be part of the appending method. I am not sure if the `buildPath` includes the debug/release components but can you make sure?

```swift
let coverageDir = buildParameters.buildPath.appending(components: "codecov", "default%m.profraw")
```

Reply:
done